packages/translate/__test__/elf-translate.navigator.test.js

```diff
@@ -8,16 +8,10 @@ describe('Elf Translate Navigator Test', () => {
 
     let el;
 
-    // IE does not support forcing of navigator language
-    if (isIE()) {
-      el = await fixture('<test-translate></test-translate>');
-    }
-    else {
-      // Force navigator language
-      Object.defineProperty(navigator, 'language', { value: 'ru' });
-      el = await fixture('<test-translate></test-translate>');
-      expect(el.defaultEl.innerText).to.equal('Региональные настройки: ru', 'Navigator locale is not taken into account');
-    }
+    // Force navigator language
+    Object.defineProperty(navigator, 'language', { value: 'ru' });
+    el = await fixture('<test-translate></test-translate>');
```

Comment:
let or const for `el` should not declare on this line

Reply:
fixed